tests/e2e-leg-9/online-upgrade-pods-pending/26-assert.yaml

```
@@ -0,0 +1,20 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-base-upgrade-sec2
+status:
+  availableReplicas: 0
+  replicas: 1
```

It does not necessarily mean the pods are pending. You need to validate the pod itself:
```suggestion
apiVersion: v1
kind: Pod
metadata:
  name: v-base-upgrade-sec2-0
status:
 phase: pending
```

---

Thanks. This is helpful.